javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,219 @@
+
+let nameArray = [];
+let todoArray = [];
+
+function getReply(command)
+{
+    let wordString = command.split(" "); 
+    let KeyWord = wordString[0];  // extract the keyword
```

Comment:
These should be `const`.

Reply:
Variable names are usually written in `camelCase`, so this would be `const keyWord`.